goon/browserassets/css/browserOutput.css

```diff
@@ -393,6 +393,7 @@ h1.alert, h2.alert		{color: #000000;}
 .redtext				{color: #FF0000;	font-size: 24px;}
 .clown					{color: #FF69Bf;	font-size: 24px; font-family: "Comic Sans MS", cursive, sans-serif; font-weight: bold;}
 .his_grace				{color: #15D512;	font-family: "Courier New", cursive, sans-serif; font-style: italic;}
+.spooky					{color: #FF6100;}
```

Comment:
You may want to make this a little darker for the sake of readability in light theme as well, but that's entirely up to you.